src/aquarium-optimized/Aquarium.cpp

```diff
@@ -204,10 +205,8 @@ bool Aquarium::init(int argc, char **argv) {
   }
   std::bitset<static_cast<size_t>(TOGGLE::TOGGLEMAX)> availableToggleBitset =
       mContext->getAvailableToggleBitset();
-  if (availableToggleBitset.test(
-          static_cast<size_t>(TOGGLE::UPATEANDDRAWFOREACHMODEL))) {
-    toggleBitset.set(static_cast<size_t>(TOGGLE::UPATEANDDRAWFOREACHMODEL));
-  }
+  mDrawPerInstance =
+      availableToggleBitset.test(static_cast<size_t>(TOGGLE::DRAWPERINSTANCE));
```

Comment:
Can we keep `toggleBitset.set()` at the same time? No matter this has any actual effect or not, without it gives me the feeling that the backend and the frontend are out of sync.

Reply:
Done